### DISCUSSION TITLE
Is manual cuDNN installation still recommended?

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#3489] [General]
Hi,

Just done a cuda and torch upgrade (I was running on cuda 11.3, I wasn't aware it's something I should be doing regularly) and found this recommendation in the wiki:

https://github.com/vladmandic/automatic/wiki/Torch-CUDA-Notes#monkey-patching

(recommended to install cuDNN manually instead of relying on whatever torch installs by default)

Is this still valid, should I (try to) perform the steps described there? Btw, I am on windows and ldconfig does not seem to exist as a valid executable.

**Comment by vladmandic** [comment]
no, that is not required for a looong time, that wiki page needs an update.
